### PR TITLE
Cortex xdr   port scan   adjusted   add input internal ranges

### PR DESCRIPTION
**Pull request by evisochek**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/30132

## Description
Edited the input description of *InternalIPRange* to indicate the deprecation for this playbook input.
Please use "InternalIPRanges" input instead.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

**Comment by ShirleyDenkberg** [comment]
@michalgold Doc review done.

File: Packs/CortexXDR/Playbooks/Cortex_XDR_-_Port_Scan_-_Adjusted.yml [hunk 1]
```diff
@@ -1127,6 +1133,13 @@ inputs:
   required: false
   description: Unique ID for the XDR alert.
   playbookInputQuery:
+- key: InternalIPRange
+  value: {}
+  required: false
+  description: |-
+    Please use "InternalIPRanges" input instead.
+    this input  deprecated.
```
Comment (michalgold) [inline Packs/CortexXDR/Playbooks/Cortex_XDR_-_Port_Scan_-_Adjusted.yml:1141]: Probably should be "This (upper case) input is deprecated". @ShirleyDenkberg @evisochek

Comment (evisochek) [reply Packs/CortexXDR/Playbooks/Cortex_XDR_-_Port_Scan_-_Adjusted.yml]: Fixed @michalgold

**Review by michalgold — APPROVED** [review]

**Comment by evisochek** [comment]
create new PR and combined the:
https://github.com/demisto/content/pull/9854
https://github.com/demisto/content/pull/9869

**Pull request closed without merging**